cli/src/factory_ops/unboxing/init

```diff
@@ -304,7 +304,16 @@ echo "Bringing HCTL cluster out of maintenance mode" 2>&1 | tee -a ${LOG_FILE}
 hctl node unmaintenance --all 2>&1 | tee -a ${LOG_FILE}
 
 echo "Ensure cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
-ensure_healthy_cluster
+
+ensure_healthy_cluster "false" "false"
```

Comment:
I think this should run in a loop, we are just trying it once. I think we should run this loop until `$ret_ensure_healthy_cluster -ne 0 `. So instead of if loop it should be a while loop.